### PR TITLE
footer-header-tablet-and-some-fixesOnMobile

### PR DESCRIPTION
**Pull request by aahland**
-små ändringar i mobile header och footer
-media query tablet size header och footer
- stylingen för header och footer är nu anpassad för mobil 375px och tablet 834px men inget däremellan så ingen smooth transition om man drar i fönstret

### CODE REVIEW

File: footer.css [hunk 1]
```diff
@@ -85,7 +85,38 @@ footer{
     margin-left:15px;
 }
 
+@media screen and (min-width: 834px) {
+    footer {
+      width: 834px;
+      height: 625px;
+    }
+    #logoAdressContactSection{
```
Comment (postmodernistx) [inline footer.css:93]: Are the ID's used for JavaScript purposes? Otherwise, change to the more re-usable class syntax :)

File: footer.css [hunk 2]
```diff
@@ -85,7 +85,38 @@ footer{
...
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        width:685px;
+        margin-top: 50px;
+        
+
```
Comment (postmodernistx) [inline footer.css:100]: Some extra line breaks here, but not on other selectors?

File: header.css [hunk 3]
```diff
@@ -1,27 +1,31 @@
...
     margin:0px;
+    
 }
 
 #fishlogoblue {
     height: 40px;
     width: 54px;
     margin-bottom: 2px;
     margin-left:10px;
+    margin-right:27px;
```
Comment (postmodernistx) [inline header.css:14]: Some inconsistencies between space after `:` - some have, others don't. Consider using a style linter in the group, to ensure a consistent style :)

File: index.html [hunk 4]
```diff
@@ -15,7 +15,6 @@
 
 <body>
     <header>
-
         <span id="logotypeHeader">
             <img id="fishlogoblue" src="img/kitchen-fish.blue.svg" alt="fish logo">
```
Comment (postmodernistx) [inline index.html:19]: Maybe the alt text could be more descriptive for the logo image?

**Review by postmodernistx — APPROVED** [review]
Overall, looks good!

If the ID CSS selectors are used for JavaScript, keep them. Otherwise change to class, as classes can be re-used across the project. Creates a more maintainable project, re-usable code as well as preps for future maintenance.

**Review by malinaline — APPROVED** [review]

File: footer.css (hunk 1 above)
Comment (aahland) [reply footer.css:93]: thanks, will change to classes instead!

**Comment by aahland** [comment]
closed - did changes and will push that code instead

**Pull request closed without merging**